typescript/agentkit/README.md

```diff
@@ -30,6 +30,9 @@ AgentKit is a framework for easily enabling AI agents to take actions onchain. I
   - [PrivyWalletProvider](#privywalletprovider)
     - [Authorization Keys](#authorization-keys)
     - [Exporting Privy Wallet information](#exporting-privy-wallet-information)
+  - [SolanaKeypairWalletProvider](#solanakeypairwalletprovider)
+    - [Network Configuration](#solana-network-configuration)
+    - [RPC URL Configuration](#rpc-url-configuration)
```

Comment:
Let's create a new section called "SVM Wallet Providers" that includes only this, and then rename the existing "Wallet Providers" to "EVM Wallet Providers"